src/sidebar/panel.js

```diff
@@ -26,14 +33,14 @@ function handleLocalContent(data) {
   if (!data.hasOwnProperty('notes')) {
     quill.setContents({
       ops: [
-        { attributes: { size: 'large', bold: true }, insert: 'Welcome!' },
-        { insert: '\n\n' },
+        { attributes: { size: 'large', bold: true }, insert: browser.i18n.getMessage('welcomeTitle') },
+        { insert: '\n\n', attributes: { direction: LANG_DIR, align: TEXT_ALIGN_DIR }},
```

Comment:
Since we're adding the direction on the newline at the end it did hold the formatting for the next new line as well as you type. There was a weird behaviour though where if you select all the text and type over it, it goes back to LTR. However, if you select all, delete text, and type it'll be good in RTL. We've added an icon so that in the last case they can just press the icon to change it back to RTL.

in the gif the first time i select all -> delete -> type
and the second time i select all -> type

![ql](https://user-images.githubusercontent.com/9160471/27442624-b8fbeda2-573e-11e7-8913-405fe2ff5e87.gif)